lib/test-worker.js

```diff
@@ -34,73 +30,40 @@ sourceMapSupport.install({
 		if (sourceMapCache[source]) {
 			return {
 				url: source,
-				map: sourceMapCache[source]
+				map: fs.readFileSync(sourceMapCache[source], 'utf8')
```

Comment:
Hmm... I'm not sure. I guess it's fine.

On Wed, Dec 30, 2015 at 7:03 PM James Talmage <notifications@github.com>
wrote:

> In lib/test-worker.js
> <https://github.com/sindresorhus/ava/pull/390#discussion_r48645161>:
>
> > @@ -34,73 +30,40 @@ sourceMapSupport.install({
> >  		if (sourceMapCache[source]) {
> >  			return {
> >  				url: source,
> > -				map: sourceMapCache[source]
> > +				map: fs.readFileSync(sourceMapCache[source], 'utf8')
>
> Where should it go?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sindresorhus/ava/pull/390/files#r48645161>.
>